### DISCUSSION TITLE
3d Lesion segmentation, loss jumping all over the place.

### DISCUSSION THREAD

**Discussion by anandpreshob** [MIC-DKFZ/nnUNet#1315] [Q&A]
Hello,
I'm trying to run 3D multiclass segmentation of lesions on whole body scans were the lesions can range from large volumes to sparse smaller blobs. Loss values for both training and validation reduces for few epochs (100) but then it jumps all the way up high continues to reduce for some epochs and keeps jumping abruptly up and down. Has some one experienced this and any suggestion on how to solve the same? Is the soft dice loss function taking the background under consideration as well when it is calculating the loss?

**Comment by FabianIsensee** [comment]
Try the `nnUNetTrainerV2_Loss_DiceCE_noSmooth` trainer
